src/components/DailyRoutinePage/AddedProductCard.js

```diff
@@ -2,23 +2,59 @@ import "./AddedProductCard.css";
 import { TiShoppingCart } from "react-icons/ti";
 import bottleimg from "../../images/niacinamide-bottle.png";
 
-export default function AddedProductCard() {
+import getProductsById from "../../utility/getProductsById";
+
+export default function AddedProductCard({ info, products, name, time }) {
+  let i;
+  if (name === "Monday") {
+    i = 0;
+  } else if (name === "Tuesday") {
+    i = 1;
+  } else if (name === "Wednesday") {
+    i = 2;
+  } else if (name === "Thursday") {
+    i = 3;
+  } else if (name === "Friday") {
+    i = 4;
+  } else if (name === "Saturday") {
+    i = 5;
+  } else if (name === "Sunday") {
+    i = 6;
+  }
+
+  const weekdayTime = info.filter((product) => {
+    return product.days[i][time];
+  });
+
+  const openingDate = info.filter((product) => {
```

Comment:
Same here, `openingDate` would be an array. Maybe there is a better name for this?
Or maybe you wanted to do something else?

Reply:
Vale, I see, naming is definitely not my strong suit :D